contrib/pai_vscode/README_zh_CN.md

```diff
@@ -25,7 +26,7 @@ OpenPAI VS Code Client 是一个 Visual Studio Code 的扩展组件，可以连
 注意， OpenPAI 集群的版本必须大于或等于 0.8.0。
 
 1. 使用快捷键 <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> 打开命令面板。
-2. 如下输入并查找 *PAI: Add PAI Cluster*。
+2. 如下输入并查找 *PAI: 添加 PAI 集群*。
```

Comment:
Can you find the command by Chinese?

Reply:
YES.
![cn](https://user-images.githubusercontent.com/17357108/75664488-82191180-5cad-11ea-9e1b-07e413f044e8.png)